blivet/populator/helpers/lvm.py

```diff
@@ -390,7 +395,8 @@ def _add_vg_device(self):
                                              pe_count=pe_count,
                                              pe_free=pe_free,
                                              pv_count=pv_count,
-                                             exists=True)
+                                             exists=True,
+                                             exported=vg_info.exported)
```

Comment:
Would it make sense to do something like `getattr(vg_info, "exported", False)` so that this can be merged right away?

Reply:
I was actually thinking about doing that, but it might be confusing to have the property set to false for exported vgs just because we have old version of libblockdev.

Reply:
Then it can be set to `None` for now which evaluates to `False` as a boolean anyway. But I see your point.

Reply:
I've set the default value to `False` added a log message that should help with debugging in case there really is an exported vg.
Thoughts @dwlehman ?